src/wallet.js

```diff
@@ -1011,6 +1027,8 @@ const wallet = {
       if (txout.spent_by !== null) {
         return {exists: false, message: `Output [${index}] of transaction [${txId}] is already spent`};
       }
+      // Set txout height as block height (if it's tx it will be undefined and the lib will handle it)
+      txout.height = tx.height;
```

Comment:
Why are settings it to the txout? Is it an internal thing?

Reply:
Yes, this method is used when I choose the inputs of a transaction, so I have to check if the output exists and return it, in case of success. This output is then checked to see if it's valid to be spent (lock checks), so I need to have the tx height of it. That's why I set the height of the output as the height of it's transaction, so I can use elsewhere.

Reply:
I got it. Maybe we should include in the documentation that the txout may have a height that is a copy of its block height. What do you think?

Reply:
I am still not sure where we should add this information. We currently don't have any centralized place to describe the objects. I've added this to the method docstring. Do you think it's enough for now, or maybe add elsewhere also?

Reply:
It seems good for now. Maybe we should create an issue to describe the objects. I think it is important to keep track of their properties and types.

Reply:
I agree. I think one of the things we should do in the lib refactor is to create classes for each of the objects we have, like a model. So it will be described and designed in the big refactor we will do

Reply:
Ok. I took note of this idea in the Issue description. https://github.com/HathorNetwork/hathor-wallet-lib/issues/41